owltorrent-network/src/main/java/edu/rice/owltorrent/network/messages/ChokeMessage.java

```diff
@@ -0,0 +1,30 @@
+package edu.rice.owltorrent.network.messages;
+
+import edu.rice.owltorrent.network.PeerMessage;
+import java.nio.ByteBuffer;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+/**
+ * Choke Message for Peer Message
+ *
+ * @author: Shijie Fan
+ */
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class ChokeMessage extends PeerMessage {
+    public static int chokeMessageSize = 5; // 4 bytes for length + 1 byte for type
```

Comment:
Make this final. Also it's good practice to have static final constants named in all caps.